k2/csrc/hash.h

```diff
@@ -1014,6 +1014,349 @@ class Hash {
 };
 
 
+/*
+  How class Hash64 works:
+
```

Comment:
The documentation looks the same as Hash.. should be changed!

Reply:
It does not document what a bucket contains and how a key-value pair is stored.